### ISSUE TITLE
Strange things happens with error box notification

### ISSUE THREAD

**Issue by xrobachok** [pressupinc/require-featured-image#12]
So, I've added featured image with correct width and height but when I were hover on editor buttons I've got an error box that says my featured image smaller than required sizes. This box appear just for 1 sec from time to time. I've recorded screen with this bug http://i.imgur.com/1Rp3seT.gifv

Can you fix it? Thank you! Wordpress 4.8.

**Comment by davidbhayes** [comment]
Hey @xrobachok, thanks for reporting. And the GIF is great! A few questions that'll help me reproduce: 

- Does this happen repeatedly (over-and-over)? Or it just after you add it at first?
- What browser(s) do you see this behavior in?

**Comment by xrobachok** [comment]
It looks like this bug exist even if I just clicked a lot of times in editor textarea, but in this case this box appear not so offten as if I just hover on editor buttons.

I've recorded to you 2 more GIFV with this bug:

1. http://i.imgur.com/xIe3TWE.gifv as you see, before page loading error box appear just 1 time
2. http://i.imgur.com/dczQCub.gifv as you see, before page loading error box appear 2 times

May be this is just conflict with some other plugins. I will try to check this only with your plugin enabled.

PS: this is affected only stored posts (like published or drafts), so if I've just click on Add new post and nothing do - there is no bug because I've already seen the box with text that I didn't setup featured image yet and that's norrmal.
